commitizen/git.py

```diff
@@ -119,19 +119,22 @@ def commit(
     f.write(message.encode("utf-8"))
     f.close()
 
-    command = f'git commit {args} -F "{f.name}"'
-
-    if committer_date and os.name == "nt":  # pragma: no cover
-        # Using `cmd /v /c "{command}"` sets environment variables only for that command
-        command = f'cmd /v /c "set GIT_COMMITTER_DATE={committer_date}&& {command}"'
-    elif committer_date:
-        command = f"GIT_COMMITTER_DATE={committer_date} {command}"
-
+    command = _get_commit_command(args, committer_date, f.name)
     c = cmd.run(command)
     os.unlink(f.name)
     return c
 
 
+def _get_commit_command(args: str, committer_date: str | None, name: str) -> str:
```

Comment:
```suggestion
def _generate_commit_command(args: str, committer_date: str | None, name: str) -> str:
```

it might be more accurate this way. We're not getting it from anywhere. (yep, but it sometimes make me feel like it's a generator...)

Reply:
sounds like a good option